packages/xod-client-electron/src/view/containers/App.jsx

```diff
@@ -33,20 +31,22 @@ import { SaveProgressBar } from '../components/SaveProgressBar';
 
 import formatError from '../../shared/errorFormatter';
 import * as EVENTS from '../../shared/events';
-import * as MESSAGES from '../../shared/messages';
 import UPLOAD_MESSAGES from '../../upload/messages';
 import { createSystemMessage } from '../../shared/debuggerMessages';
 
 import { subscribeAutoUpdaterEvents } from '../autoupdate';
 import { subscribeToTriggerMainMenuRequests } from '../../testUtils/triggerMainMenu';
 
+import { getOpenDialogFileFilters, createSaveDialogOptions } from '../dialogUtils';
```

Comment:
Too many utils. XOD as the whole is util :smile: Maybe, `dialogUtils` → `nativeDialogs`?